example/ndpiReader.c

```diff
@@ -709,8 +709,8 @@ static void help(u_int long_help) {
            sizeof(((struct ndpi_flow_struct *)0)->protos));
 
     printf("\n\nnDPI supported protocols:\n");
-    printf("%3s %8s %-22s %-10s %-8s %-12s %s\n",
-	   "Id", "Userd-id", "Protocol", "Layer_4", "Nw_Proto", "Breed", "Category");
+    printf("%3s %8s %-22s %-10s %-8s %-12s %-18s %-20s %-20s \n",
```

Comment:
18,25,25
Since there is at least one protocol with 5 ports, I think we can use the max value (+ 1 as space)

Reply:
Done. I agree with you